src/App.tsx

```diff
@@ -1,63 +1,145 @@
+      newStack.push(newMap);
+      setWorld(newStack);
     }
-  };
+  }
+
+  //TODO: Change from alert to something else
+  const confirmChoice = (choice: string) => {
+    alert("Is " + choice + " your final answer?")
+  }
+
+  const evaluateChoice = (choice: string) => {
+    
+    setWorld(new Stack<MapDetails>(5, AllWorlds));
+    setVideoShown(true);
+    displayFeedback(choice === currentQuestion.answerName);
+
+    setTimeout(() => {
+        setResult(undefined);
+        setVideoShown(false);
+        setHoverArea("");
+        setCurrentQuestion(getNextQuestion());
+    }, 10000);
+  
+  }
+
+  const displayFeedback = (result: Boolean) => {
+    setResult(result);
+    if (result) setScore(score + 1);
+    // if (score === 5) //TODO: End Game
+    
+  }
+
+  const enterArea = (area: MapAreas) => {
+    setHoverArea(area.id);
+  }
+
+  const leaveArea = () => {
+    setHoverArea("");
+  }
 
   return (
     <div className="App">
       <Header />
-      <BackgroundImage imageUrl="./wow-map-min.jpg" />
-      <Grid container spacing={3} justifyContent="center" alignItems="center">
-        {currentQuestion.options.map((q, i) => (
-          <Grid item xs={12}>
-            <Card sx={{ minWidth: 275 }}>
-              <CardActionArea onClick={() => onQuestionSelect(q)}>
-                <CardContent>
-                  <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
-                    Option #{i + 1}
-                  </Typography>
-                  <Typography variant="h5" component="div">
-                    {worldOfWarcraft[q]?.name}
-                  </Typography>
-                </CardContent>
-              </CardActionArea>
-            </Card>
-          </Grid>
-        ))}
-      </Grid>
-      <div id="MainContainer">
-        <YoutubePlayer />
+
+      {(videoShown === true)
+      ? result===true
+        ?<h3>Correct!</h3>
+        :<h3>Incorrect!</h3>
+      :<div className="map-area">
+        <div className="presenter">
+          <div className="photo">
+            <ImageMapper {...imageMapperProps} />
+            <Typography
+              variant="h6"
+              className="you-text"             
+              sx={{
+                mr: 2,
+                display: { xs: 'none', md: 'flex' },
+                fontFamily: 'monospace',
+                fontWeight: 700,
+                letterSpacing: '.3rem',
+                color: 'white',
+                textDecoration: 'none',
+              }} 
+            >
+            {hoverArea}
+            </Typography>
+          </div>
+        </div>
+      </div>
+      }
+
+      <YoutubePlayer />
+
+      <div className="map-controls">
+        <Button 
+          className="back-button" 
+          disabled={!!videoShown || world.peek() === AllWorlds} 
+          onClick={() => { removeMap(); }} 
+          variant="contained" 
+          startIcon={<ReplayIcon />}
+        >
+        Back One Map
```

Comment:
Maybe we can just update this to `back`? I think it'd be better to move this button to the top left of the map image. The location for me is a bit confusing.

Reply:
Or even could just be a back arrow in the top left corner of the Map Image?